src/main/java/model/StoreOperation.java

```diff
@@ -0,0 +1,35 @@
+package model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum StoreOperation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private static final Map<String, StoreOperation> OPERATION_BY_LETTER = new HashMap<>();

```

Comment:
```suggestion
    private static final Map<String, StoreOperation> OPERATION_BY_CODE = new HashMap<>();
```
Operation with a two-letters code is possible in the future, why not?